scss/components/_video-quality.scss

```diff
@@ -42,23 +44,32 @@
         }
 
         //vid quality btns - full screen
-        a.oo-quality-btn {
+        .oo-quality-btn {
           padding: 10px 8px;
           background-color: transparent;
           color: $white;
+          display: inline;
           font-size: 1.1em;
           font-family: $font-family-roboto;
-          line-height:1em;
+          line-height: 1em;
           border-radius: 2px;
           border: 1px solid $gray;
           outline: none;
-          opacity:1;
+          opacity: 1;
           @include vendor-prefixes(transition, opacity 0.3s ease);
 
           &:hover {
             opacity:0.6;
           }
 
+          &:focus {
```

Comment:
Do we need spaces after the '&:' since you fixed then in the section above?

Reply:
Nope, these are usually written without spaces